### PR TITLE
added tooltip bucket

### PR DESCRIPTION
**Pull request by mvgaliev**

### CODE REVIEW

File: capabilities.json [hunk 1]
```diff
@@ -63,6 +63,12 @@
             "displayName": "Extra information",
             "displayNameKey": "Role_ExtraInformation",
             "kind": "Grouping"
+        },
+        {
+            "name": "Tooltips",
+            "displayName": "Tooltips",
+            "displayNameKey": "Role_Tooltips",
+            "kind": "Measure"
```
Comment (ignatvilesov) [inline capabilities.json]: Have you tried to use `GropingAndMeasure`? Is that possible to allow users to use text data fields as well in tooltips?
IMHO text fields make sense.

File: capabilities.json [hunk 2]
```diff
@@ -100,6 +106,9 @@
                     "ExtraInformation": {
                         "min": 0,
                         "max": 3
+                    },
+                    "Tooltips": {
```
Comment (ignatvilesov) [inline capabilities.json]: There's no need to add any conditions for `Tooltips` field.

File: capabilities.json [hunk 3]
```diff
@@ -63,6 +63,12 @@
             "displayName": "Extra information",
             "displayNameKey": "Role_ExtraInformation",
             "kind": "Grouping"
+        },
+        {
+            "name": "Tooltips",
+            "displayName": "Tooltips",
+            "displayNameKey": "Role_Tooltips",
```
Comment (ignatvilesov) [inline capabilities.json]: `Role_Tooltips` must be included to [resources.resjson](https://github.com/Microsoft/powerbi-visuals-gantt/blob/master/stringResources/en-US/resources.resjson) as well

File: capabilities.json (hunk 1 above)
Comment (mvgaliev) [reply capabilities.json]: It is possible to use text data for tooltips. Not really sure why we need to use GropingAndMeasure in this case. We need a specific condition to use GropingAndMeasure

File: capabilities.json (hunk 3 above)
Comment (mvgaliev) [reply capabilities.json]: added

**Pull request merged**